service/publish/model/publish.go

```diff
@@ -0,0 +1,12 @@
+package model
+
+import "gorm.io/gorm"
+
+// Publish 视频表 /*
+type Publish struct {
+	gorm.Model
```

Comment:
Please move this model definition to repo/model & use model.Model as the base model.

Reply:
Also, you might need to add your model to [migration](https://github.com/Toktik-Team/toktik/blob/a08cafc7e836c830921b7b8ed82b6c8cbf9626c7/repo/migrate/migrate.go#L24) and [codegen](https://github.com/Toktik-Team/toktik/blob/a08cafc7e836c830921b7b8ed82b6c8cbf9626c7/repo/gen/gormGen.go#L28) scripts.

Reply:
Sorry, this is a mistake. It should be deleted.